src/main/java/net/dv8tion/jda/core/JDA.java

```diff
@@ -208,6 +210,24 @@ public boolean equals(Object o)
     User getUserById(String id);
 
     /**
+     * Gets all {@link Guild}s that contain all given users as their members.
+     *
+     * @param users The users which all the returned {@link Guild}s must contain.
```

Comment:
The description of a parameter needs to be aligned in the next line as you can see in the javadoc template.
Same for docs below.